handbook/product/rollout_process.md

```diff
@@ -0,0 +1,47 @@
+# Rollout Process
+
+The rollout process consists of *Pre-Launch* phase and *Deployment* phase. We can push changes to Sourcegraph.com when we
+want as we own the deployment process. For enterprise instances of Sourcegraph.com, we push a new release monthly and our
+clients decide to upgrade to the latest release when they want.
+
+## Sourcegraph.com
+
+### Pre-Launch
+**User Testing** 
+- Hallway test with internal users.
+- Open the feature to internal users by turning it on for the Sourcegraph organization. Make sure it's opened with enough
+time left before planned launch date to receive and address all the feedback received.  Follow this [document](https://about.sourcegraph.com/handbook/engineering/distribution/update_sourcegraph_website) to enable your feature in site settings and turn it on only for internal users in the Sourcegraph organization. TODO: Update doc linked above to add a bit more content for new teammates
```

Comment:
Turning it on in site settings turns it on for everyone, it needs to be set in org settings.
```suggestion
time left before planned launch date to receive and address all the feedback received.  Go to [our Sourcegraph organization settings](https://sourcegraph.com/organizations/sourcegraph/settings) to enable your feature only for internal users in the Sourcegraph organization.
```

Reply:
Added a test search scope to test this quickly myself too. Thanks for fixing this!